tests/unit/aiplatform/test_automl_tabular_training_jobs.py

```diff
@@ -252,6 +252,12 @@ def mock_dataset_tabular():
     )
     ds.column_names = _TEST_TRAINING_COLUMN_NAMES
 
+    # TODO: Do this correctly
```

Comment:
@sasha-gitg Not sure how to mock this method correctly.

I've seen `patch` used in other places.

Reply:
In any case, using `mock_dataset_tabular` prevents the unit test from actually testing if TabularDataset has a `column_names` method or not.

How should I rectify this?

Reply:
Removed